pkgs/development/python-modules/pymilter/default.nix

```diff
@@ -14,29 +13,22 @@
 
 buildPythonPackage rec {
   pname = "pymilter";
-  version = "1.0.5";
+  version = "1.0.6";
   format = "setuptools";
 
   src = fetchFromGitHub {
     owner = "sdgathman";
     repo = pname;
     rev = "${pname}-${version}";
-    hash = "sha256-gZUWEDVZfDRiOOdG3lpiQldHxm/93l8qYVOHOEpHhzQ=";
+    hash = "sha256-plaWXwDAIsVzEtrabZuZj7T4WNfz2ntQHgcMCVf5S70=";
   };
 
   buildInputs = [ libmilter ];
   nativeCheckInputs = [ pyasyncore ];
   propagatedBuildInputs = [
-    bsddb3
+    berkeleydb
     pydns
   ];
-  patches = [
-    (fetchpatch { # https://github.com/sdgathman/pymilter/pull/57
-      name = "Remove-calls-to-the-deprecated-method-assertEquals";
-      url = "https://github.com/sdgathman/pymilter/commit/1ead9028fc63ae3ec6ea3b0c438e6ed088a2b20e.patch";
-      hash = "sha256-/5LlDR15nMR3l7rkVjT3w4FbDTFAAgNdERWlPNL2TVg=";
-    })
-  ];
 
   preBuild = ''
     sed -i 's/import thread/import _thread as thread/' Milter/greylist.py
```

Comment:
I don't think this is needed anymore: https://github.com/sdgathman/pymilter/blob/pymilter-1.0.6/Milter/greysql.py#L8